hw/ip/aon_timer/README.md

```diff
@@ -14,7 +14,7 @@ See that document for an overview of how it is integrated into the top level sys
 
 ## Features
 
-- Two 32-bit upcounting timers: one timer functions as a wakeup timer, one as a watchdog timer
+- Two upcounting timers: one 64-bit timer functions as a wakeup timer, and a 32-bit one as a watchdog timer
```

Comment:
Silly grammar nit: the sentence doesn't quite work (and didn't before your change either!) How about:

```
- Two upcounting timers: one 64-bit timer that functions as a wakeup timer, and a 32-bit one that functions as a watchdog timer
```
?